diff --git a/app/boards/jakkra/zswatch/zswatch_nrf5340_cpuapp_5.overlay b/app/boards/jakkra/zswatch/zswatch_nrf5340_cpuapp_5.overlay
index 32d0880..2b27718 100644
--- a/app/boards/jakkra/zswatch/zswatch_nrf5340_cpuapp_5.overlay
+++ b/app/boards/jakkra/zswatch/zswatch_nrf5340_cpuapp_5.overlay
@@ -200,7 +200,7 @@
 		reg = <0x51>;
 		status = "okay";
 		clkout = <0>;
-		int-gpios = <&gpio1 13 (GPIO_PULL_UP)>;
+		int-gpios = <&gpio1 13 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
 	};
 
 	bmi270: bmi270@68 {
@@ -265,11 +265,11 @@
 
 		npm1300_charger: charger {
 			compatible = "nordic,npm1300-charger";
-			term-microvolt = <4400000>;
+			term-microvolt = <4200000>;
 			term-warm-microvolt = <4000000>;
-			current-microamp = <200000>; // 0.2C CC (constant current) charge to 4.4V,
-			term-current-percent = <10>; // then 4.4V CV(constant voltage) charge till charge current decline to ≤ 0.02C
-			dischg-limit-microamp = <1000000>;
+			current-microamp = <32000>; // 0.2C CC (constant current) charge to 4.2V,
+			term-current-percent = <10>; // then 4.2V CV(constant voltage) charge till charge current decline to ≤ 0.01C
+			dischg-limit-microamp = <200000>;
 			vbus-limit-microamp = <500000>;
 			thermistor-ohms = <0>; // Don't use thermistor
 			thermistor-beta = <0>; // Not used
